app/controllers/api/v2/hosts_controller.rb

```diff
@@ -197,6 +197,26 @@ def get_status
         end
       end
 
+      api :DELETE, "/hosts/:id/status/:type", N_("Clear sub-status of host")
+      param :id, :identifier_dottable, :required => true
+      param :type, HostStatus.status_registry.to_a.map { |s| s.humanized_name }, :required => true, :desc => N_(
+        <<~EOS
+          status type
+        EOS
+      )
```

Comment:
@ianballou was there any reason you didn't use 
```suggestion
      param :type, HostStatus.status_registry.to_a.map { |s| s.humanized_name }, :required => true, :desc => N_("status type"))
```
here?